jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -174,10 +174,11 @@ def test_moment_velocity_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
     mm.calculate_moment()
 
     label_mouseover._viewer_mouse_event(uncert_viewer, {'event': 'mousemove',
-                                                        'domain': {'x': 0, 'y': 0}})
-    assert label_mouseover.as_text() == ("Pixel x=00.0 y=00.0 Value -2.99792e+08 m / s",
-                                         "World 13h39m59.9731s +27d00m00.3600s (ICRS)",
-                                         "204.9998877673 27.0001000000 (deg)")
+                                                        'domain': {'x': 1, 'y': 1}})
```

Comment:
Why change it to `x=1 y=1`? Hard to compare the diff to see if results are the same or not before/after patching.

Reply:
The new moment 2 values are indeed different, that calculation was incorrect before. And I switched to (1,1) because the new value at 0 is 0, which is boring 🙂 .

Reply:
That different, huh? I wonder how many scientists trusted the wrong values blindly. 😱 

What about the moment 1 differences above?